app/containers/Certifier/CertificationRequestsContainer.tsx

```diff
@@ -136,6 +136,7 @@ export const CertificationRequestsComponent: React.FunctionComponent<Props> = ({
               <label>
                 {!node.certifiedAt && (
                   <input
+                    aria-label="Select for certification"
```

Comment:
Violation: Form inputs must have labels

Fix: Using an invisible `aria-label`, as this is not a form but an interactive control element to select applications for certification. The context is conveyed by the "Select All" label at the top of the column and it would be redundant or confusing to add a visual "Select" label for each row in the table.